storage/people.json

```diff
@@ -0,0 +1 @@
+[{"id":425,"age":14,"name":"red","rentals":[]},{"id":152,"age":15,"name":"sane","rentals":[]},{"id":978,"age":54,"name":"red","rentals":[]},{"id":474,"age":16,"name":"guy","rentals":[]}]
```

Comment:
- [Optional] Data being saved at the end when I exit from the app doesn't contain properties for both Teacher and Student. parent_permission property from the student and specialization property from the teacher is missing in person.json file.  I think all the properties of both teacher and student should be written in this file at the end

Reply:
Hello @FrederickMih, thank you for your kind suggestion. I will try to add these properties.